enterprise/fulltext-addon/src/main/java/org/neo4j/kernel/api/impl/fulltext/integrations/bloom/BloomProcedures.java

```diff
@@ -57,27 +57,37 @@ public void awaitPopulation() throws IOException
 
     @Description( "Queries the bloom index for nodes" )
     @Procedure( name = "db.fulltext.bloomFulltextNodes", mode = READ )
-    public Stream<Output> bloomFulltextNodes( @Name( "terms" ) List<String> terms ) throws Exception
+    public Stream<Output> bloomFulltextNodes( @Name( "terms" ) List<String> terms, @Name( "fuzzy" ) boolean fuzzy,
+            @Name( "Require that all terms are matched" ) boolean matchAll ) throws Exception
     {
         try ( ReadOnlyFulltext indexReader = provider.getReader( BLOOM_NODES, FulltextProvider.FulltextIndexType.NODES ) )
         {
-            return queryAsStream( terms, indexReader );
+            return queryAsStream( terms, indexReader, fuzzy, matchAll );
         }
     }
 
     @Description( "Queries the bloom index for relationships" )
     @Procedure( name = "db.fulltext.bloomFulltextRelationships", mode = READ )
-    public Stream<Output> bloomFulltextRelationships( @Name( "terms" ) List<String> terms ) throws Exception
+    public Stream<Output> bloomFulltextRelationships( @Name( "terms" ) List<String> terms, @Name( "fuzzy" ) boolean fuzzy,
+            @Name( "Require that all terms are matched" ) boolean matchAll ) throws Exception
```

Comment:
You could set `defaultValue`s for the `fuzzy` and `matchAll` parameters. Just a thought.